reinforcement_learning/rlclientlib/ranking_event.h

```diff
@@ -33,27 +41,30 @@ namespace reinforcement_learning {
     ranking_event(ranking_event&& other);
     ranking_event& operator=(ranking_event&& other);
 
-    virtual void serialize(utility::data_buffer& buffer) override;
-  
+    virtual flatbuffers::Offset<RankingEvent> serialize_eventhub_message(flatbuffers::FlatBufferBuilder& builder);
+
   public:
     static ranking_event choose_rank(utility::data_buffer& oss, const char* event_id, const char* context,
       unsigned int flags, const ranking_response& resp, float pass_prob = 1);
 
   private:
-    ranking_event(const char* event_id, float pass_prob, const std::string& body);
+    ranking_event(const char* event_id, float pass_prob, const char* context, const ranking_response& response);
 
-  private:
-    std::string _body;
+    std::string _context;
+    std::vector<uint64_t> _a_vector;
```

Comment:
can you please make these names clearer?

Reply:
These are exact names "a", "p" we are using in ds json. Changing to other name will cause more confusion.

Reply:
See previous definition before flatbuffer:
R"({"Version":"1","EventId":"event_id","a":[],"c":{context},"p":[],"VWState":{"m":"model_id"}})"

Reply:
Please add a comment then, the ds json isn't exactly self explanatory either